slasher/beaconclient/receivers.go

```diff
@@ -70,9 +83,16 @@ func (bs *Service) receiveAttestations(ctx context.Context) {
 			log.WithError(ctx.Err()).Error("Context canceled - shutting down attestations receiver")
 			return
 		}
+		if err != nil && strings.Contains(strings.ToLower(err.Error()), strings.ToLower(context.Canceled.Error())) {
```

Comment:
+1

Reply:
+1

Reply:
fixed and tested by following https://stackoverflow.com/questions/52969205/how-to-assert-grpc-error-codes-client-side-in-go